cluster.go

```diff
@@ -711,12 +835,29 @@ func (c *clusterClient) _pickMultiCache(multi []CacheableTTL) *connretrycache {
 	defer c.mu.RUnlock()
 
 	count := conncountp.Get(len(c.conns), len(c.conns))
-	for _, cmd := range multi {
-		p := c.slots[cmd.Cmd.Slot()]
-		if p == nil {
-			return nil
+	if c.opt.SendToReplicas == nil {
+		for _, cmd := range multi {
+			p := c.pslots[cmd.Cmd.Slot()]
+			if p == nil {
+				return nil
+			}
+			count.m[p]++
+		}
+	} else {
+		for _, cmd := range multi {
+			var p conn
+			isSendToReplicas := c.opt.SendToReplicas(Completed(cmd.Cmd))
+			if isSendToReplicas {
```

Comment:
```suggestion
			if c.opt.SendToReplicas(Completed(cmd.Cmd)) {
```

Reply:
[600ecb0](https://github.com/redis/rueidis/pull/402/commits/600ecb060dcf12a25048d89d10783a8598158770)

also change it.